data/names.txt

```diff
@@ -265,13 +308,24 @@ phrase "nouns of indeterminate quantity"
 		"Mirth"
 		"Olympus"
 		"Light"
+		"Charity"
+		"Mercy"
+		"Hope"
+		"Virtue"
+		"Fortitude"
+		"Enlightenment"
+		"Might"
+		"Destiny"
+		"Grass"
 
 phrase "the-nouns"
 	word
 		"the "
 	phrase
 		"singular nouns"
+		"plural nouns"
 		"space nouns"
+		"cardinal directions"
```

Comment:
No. There is a 1 in 4 chance that "cardinal directions" will appear in "the-nouns", which themselves only have a 1 in 4 chance to appear in "things you can be of", which appear in 3 out of 12 name styles. So the actual chance of getting a cardinal direction in a ship name is 0.25^3, or 1.56%.